src/ol/render/canvas.js

```diff
@@ -333,7 +333,8 @@ export const measureTextHeight = (function() {
 export function measureTextWidth(font, text) {
   const measureContext = getMeasureContext();
   if (font != measureFont) {
-    measureContext.font = measureFont = font;
+    measureContext.font = font;
+    measureFont = measureContext.font;
```

Comment:
Are we doing this because the getter doesn’t always return what we set above?  This is very subtle.

Somewhat unrelated, but I’d be in favor of enforcing https://eslint.org/docs/rules/no-multi-assign (I know it is a stylistic preference).

Reply:
> Are we doing this because the getter doesn’t always return what we set above? This is very subtle.

Yes indeed. Similar to what I recently encountered in 3de05c249bc761e7e3b0193c7e08f94e12110885.

> Somewhat unrelated, but I’d be in favor of enforcing https://eslint.org/docs/rules/no-multi-assign (I know it is a stylistic preference).

Agreed. I can create a follow-up pull request with that.